static/app/components/seenByList.tsx

```diff
@@ -64,7 +64,7 @@ const SeenByList = ({
       />
       <IconWrapper iconPosition={iconPosition}>
         <Tooltip title={iconTooltip}>
-          <IconShow size="sm" color="gray200" />
+          <IconShow size="sm" color="subText" />
```

Comment:
Before
<img width="119" alt="seenByList-b" src="https://user-images.githubusercontent.com/44172267/144330608-0d3e2dfb-f508-4ec9-aecd-50bf98782a85.png">

After:
<img width="119" alt="seenByList-a" src="https://user-images.githubusercontent.com/44172267/144330600-71454261-33a4-4ff0-b4eb-213d469a1b54.png">

Reply:
What's it look like in light mode?

Reply:
<img width="143" alt="Screen Shot 2021-12-01 at 4 15 38 PM" src="https://user-images.githubusercontent.com/44172267/144334752-8640459f-6605-4801-9df0-2961d1c0c873.png">